packages/appstore/src/components/wallet-header/wallet-header-title.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Text, Badge } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { TWalletAccount } from 'Types';
+
+type TWalletHeaderTitle = Pick<TWalletAccount, 'is_demo' | 'currency' | 'landing_company_name'>;
+
+const WalletHeaderTitle = ({ is_demo, currency, landing_company_name }: TWalletHeaderTitle) => {
+    return (
+        <div className='wallet-header__description-title'>
+            <Text weight='bold' size='sm'>
+                {is_demo && <Localize i18n_default_text='Demo {{currency}} Wallet' values={{ currency }} />}
+                {!is_demo && <Localize i18n_default_text='{{currency}} Wallet' values={{ currency }} />}
```

Comment:
`nitpick`: can we do it like this instead

```suggestion
                {is_demo  ?
                       <Localize i18n_default_text='Demo {{currency}} Wallet' values={{ currency }} />
                       :  <Localize i18n_default_text='{{currency}} Wallet' values={{ currency }} />
                }
```

Reply:
good point